.vscode/settings.json

```diff
@@ -14,5 +14,6 @@
     },
     "githubPullRequests.ignoredPullRequestBranches": [
         "master"
-    ]
+    ],
+    "dotnet.defaultSolution": "disable"
```

Comment:
Any reason behind changing this.

Reply:
Not sure how that got in but change has been reverted.